oojs.js

```diff
@@ -10,3 +10,27 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+ class Playlist {
+   constructor (title, song){
+     this.title= title;
+     this.song  = song;
+     this.favorites = []
+   }
+
+   addSong (song){
+     this.favorites.push (song)
```

Comment:
Instructions say that this method is supposed to add a song to the playlist's `songs` property - not `favorites`